books/free-programming-books-hi.md

```diff
@@ -8,3 +8,9 @@
 * [C++ Brief Notes \| Hindi](https://ehindistudy.com/2020/12/01/cpp-notes-in-hindi/) - Yugal Joshi
 * [C++ in Hindi \| Hindi](https://www.bccfalna.com/IOC-AllEBooks/CPPinHindi.pdf) - Kuldeep Chand (PDF)
 * [C++ Introduction Book \| Hindi](https://ncsmindia.com/wp-content/uploads/2012/04/c++-hindi.pdf) - NCMS India (PDF)
+
+
+### <a id="python"></a>Python
+
+* [Python in Hindi](https://www.khansir.co.in/python-notes-in-hindi-pdf) - Kuldeep Chand
```

Comment:
is this an authorized distribution?

Reply:
Yes, the website is a trusted platform, contains other study materials along side. And has python book inlcuded in it.